src/components/LoanCollections/KivaClassicLoanCategorySelector.vue

```diff
@@ -1,9 +1,10 @@
 <template>
 	<div
-		class="tw-flex tw-justify-center tw-max-w-[700px] tw-flex-wrap tw-mx-auto"
+		class="tw-flex tw-justify-start md:tw-justify-center
+			md:tw-flex-wrap tw-mx-auto tw-overflow-x-auto md:tw-overflow-x-none"
```

Comment:
Converts the mobile view to a horizontally scrollable row instead of stacked links.